src/js/cookieMessage.js

```diff
@@ -2,6 +2,45 @@ class CookieMessage {
 
 	constructor (CookieMessageEl) {
 		this.CookieMessageEl = CookieMessageEl;
+
+		return this.setup();
+	}
+
+	setup () {
+		const message = document.querySelector('.o-cookie-message');
+
+		if (userHasConsentedToCookies()) {
+			hideMessage();
+			return;
+		}
+
+		setupMessage();
+
+		function setupMessage () {
+			document.querySelector('.o-cookie-message__close-btn').addEventListener('click', flagUserAsConsentingToCookies);
```

Comment:
same as before, could you use data attribues? in this case something like `data-o-cookie-message-close` could work